ci-operator/config/redhat-developer/odo/redhat-developer-odo-main__odo-ocp4.16-lp-interop.yaml

```diff
@@ -1,6 +1,6 @@
 base_images:
   cli:
-    name: "4.16"
+    name: "4.15"
```

Comment:
I think this is no longer necessary for odo. I had pushed some changes in the build_root image for odo to use RHEL 9 as the base image - see https://github.com/redhat-developer/odo/pull/7243

Now, the latest run of the interop tests is fine. Well, they are still failing but due to some flaky odo tests instead, not because of missing `GLIBC` libraries. See https://prow.ci.openshift.org/view/gs/test-platform-results/logs/periodic-ci-redhat-developer-odo-main-odo-ocp4.16-lp-interop-odo-scenario-aws/1793348683945742336

So to me, this PR is not needed for odo. WDYT?

Reply:
Ack, closing this PR. thanks